packages/gatsby/src/db/nodes.ts

```diff
@@ -51,8 +54,9 @@ async function loadNodeContent(node) {
   return content
 }
 
-module.exports = {
+const nodeStore = {
   ...nodesDb,
   runQuery: runFastFiltersAndSort,
-  loadNodeContent,
```

Comment:
This isn't going to be exported, because you've overwritten the export. The best way to do this is to replace this whole thing with 
```typescript
export * from "../redux/nodes"
export { runFastFiltersAndSort as runQuery } from "../redux/run-fast-filters"
```

Reply:
I'll take your word for it but I'm wondering why tests are fine (the CI failures are unrelated as far as I can see). This suggestion caused other problems when I tried it.

I'll take another look. Thanks

Reply:
Your suggestion was correct and it seems to hold with typecheck and tests. Also allows me to remove more code and simplify the file even further. Thank you :)